addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -78,3 +85,205 @@ def _get_default_payment_method_id(self):
+
+        :returns: Stripe Secret Key
+        :rtype: str
+        """
+        return self.stripe_secret_key
+
+    def _get_stripe_webhook_secret(self):
+        """Gets Stripe Webhook Secret
+
+        :returns: Stripe Webhook Secret
+        :rtype: str
+        """
+        return self.stripe_webhook_secret
+
+    def _additional_header(self, endpoint):
+        """Returns an additional header to Stripe API calls
+
+        This method adds the Stripe Account on behalf of which the Stripe call is made.
+
+        :param endpoint: the Stripe endpoint which will be called
+        :returns: An additionnal header dict
+        :rtype: dict
+        """
+        return {}
+
+    def _get_webhook_url(self):
+        return self.company_id.get_base_url() + '/payment/stripe/webhook'
+
+    # -------------------------------------------
+    # Stripe Create Webhook
+    # -------------------------------------------
+    def action_stripe_create_webhook(self):
+        self.ensure_one()
+        if self.stripe_secret_key and not self.stripe_webhook_secret:
+            return self._create_webhook() or True
+        return True
+
+    def _create_webhook(self):
+        """Create a webhook using the Stripe API
+        """
+        if self.stripe_webhook_secret:
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _('You already have defined a Stripe Webhook.'),
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        if 'localhost' in self.company_id.get_base_url():
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _('Your URL should be publicly available in order to create a Stripe Webhook.'),
+                    'sticky': False,
+                    'type': 'danger',
+                }
+            }
+        webhook = self._stripe_make_request(
+            'webhook_endpoints',
+            payload={
+                'url': self._get_webhook_url(),
+                'enabled_events[]': WEBHOOK_HANDLED_EVENTS,
+                'api_version': API_VERSION,
+            }
+        )
+        self.stripe_webhook_secret = webhook.get('secret')
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    def action_stripe_onboarding_account(self):
+        stripe_onboarding_url = self._onboarding_url()
+        if not stripe_onboarding_url:
+            raise UserError(_("Unable to create a Stripe Account directly from your instance for the moment. Please create one from Stripe Website"))
+        return {
+            'type': 'ir.actions.act_url',
+            'url': stripe_onboarding_url,
+            'target': 'self',
+        }
+
+    def _onboarding_url(self):
```

Comment:
Regarding `_get_stripe_account_link`, I think it's better to divide and conquer, since we'll have to use only the `_get_stripe_account_link` method when we have the account_id in the SaaS module (StripeController:_stripe_onboarding_refresh()).